packages/common/scss/leaders.scss

```diff
@@ -2,6 +2,15 @@
   &__logo {
     max-width: 100%;
   }
+  &__callout {
+    align-items: center;
+    font-size: 1.25rem;
+    text-align: left;
+    div {
```

Comment:
This will effect _all_ descendent divs. Create a new element (see above)...

```scss
&__callout {
  align-items: center;
  font-size: 1.25rem;
  text-align: left;
}

&__suppliers-callout {
  font-size: 1.5rem;
  font-weight: 600;
}
```